test-lib.sh

```diff
@@ -36,7 +36,8 @@ function ct_cleanup() {
     docker stop "$container"
 
     # Container has not been removed by `docker stop` and still exists
-    if [ $( docker ps -a -f id=$container | wc -l ) -eq 2 ]; then
+    # shellcheck disable=SC2046
```

Comment:
Why are we disabling the check instead of using double quotes as expected?

Reply:
I don't like double quotes twice. :( But I can change it.

Reply:
Like here: https://github.com/sclorg/container-common-scripts/blob/master/test-lib.sh#L79